main.nf

```diff
@@ -373,6 +397,40 @@ if( params.readPaths ){
 
 }
 
+// Setup for Pop Gen
```

Comment:
This kind of set up is not the regular way to do things in Nextflow, as it hides quite some complexity that one could easily miss and then wonder whats going on. The typicaly way would be:

- adding some parameters e.g. turning on sections and then using the `when: params.foo` to turn on a module. Also using `channel.mix(...)` as a directive to get output passed along if there are Switches. 

As the Haplotypecaller isn't entirely done (missing some parts from EAGER1, e.g. the GATK best practice part is not entirely there), I would also remove this from this PR and only add one piece at a time in a PR.